plugins/sensors/switchbotmeterbt

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+
+"""
+=head1 NAME
+
+  switchbotmeterbt - Munin plugin to monitor temperature/humidity with SwitchBot Meter BLE
+
+=head1 CONFIGURATION
+
+  Python 3 and bluepy are necessary.
+
+  Example for Ubuntu:
+    $ sudo apt install python3-pip
+    $ sudo pip3 install bluepy
+
+=head1 ENVIRONMENT VARIABLES
+
+  user            : root privilege is necessary for BLE scan
+  env.macaddr     : Mac Address(es) of SwitchBot Meter(s) separated by white-space (Required)
+  env.tempunit    : Temperature unit. (Optional, default is C)
+  env.scantimeout : Timeout for BLE scan. (Optional, default is 5.0 seconds)
+
+  Example:
+    [switchbotmeterbt]
+      user  root
+      env.macaddr  aa:aa:aa:aa:aa:aa bb:bb:bb:bb:bb:bb cc:cc:cc:cc:cc:cc
+
+=head1 NOTES
+
+  For more details about SwitchBot Mater, see https://www.switch-bot.com/products/switchbot-meter
+
+=head1 AUTHOR
+
+  K.Cima https://github.com/shakemid
+
+=head1 LICENSE
+
+  GPLv2
+  SPDX-License-Identifier: GPL-2.0-only
+
+=head1 Magic markers
+
+  #%# family=contrib
+  #%# capabilities=
+
+=cut
+"""
+
+import sys
+import os
+from bluepy.btle import Scanner, DefaultDelegate
+
+
+class SwitchbotScanDelegate(DefaultDelegate):
+    def __init__(self, macaddrs):
+        DefaultDelegate.__init__(self)
```

Comment:
I think, `super().__init__()` is the modern approach for calling parent methods.

Reply:
Thanks! I am not sure which is better, but in the sample code of bluepy, `DefaultDelegate.__init__(self)` is used. What do you think?
https://ianharvey.github.io/bluepy-doc/scanner.html

Reply:
The `super` call is preferable.
A longer explanation can be found [here](https://stackoverflow.com/a/33469090).
Feel free to suggest this change to the maintainer of the above example code :)

Reply:
Thank you! I would like to fix it.